resources/config.py

```diff
@@ -42,35 +45,48 @@ def get(self):
                 description: Static configuration global statistics
                 content:
                   application/json:
-                    schema: ConfigStatsSchema
+                    schema: ConfigStatsResponseSchema
         """
-        from_time = request.args.get('range', '*')
+        schema = ConfigStatsRequestSchema()
+        params = schema.load(request.args)
+
+        if params and params.errors:
+            return {"errors": params.errors}, 400
+
+        from_time = params.data["range"]
         if from_time.endswith("h"):
             from_time = int(from_time[:-1])
         elif from_time.endswith("d"):
             from_time = int(from_time[:-1]) * 24
+        elif from_time != "*":
+            raise BadRequest("Wrong range format")
```

Comment:
What if I pass `"asdasda"` as `from_time`? It's not `"*"`, so this check will pass, but later `timedelta(hours=from_time)` will fail.

Reply:
Isn't this the other way round? `"asdasda" != "*"` so it will throw the bad request

Reply:
hmmmm, you're right.